File: Models/Responses/Budgets/ResponseCountryBusinessManager.cs
namespace Models.Responses.Budgets;

/// <summary>
/// CBM 관리 응답 모델 
/// </summary>
public class ResponseCountryBusinessManager : ResponseCommonWriter
{
    /// <summary>
    /// 아이디 
    /// </summary>
    public Guid Id { get; init; }
    
    /// <summary>
    /// 오너명
    /// </summary>
    public required string Name { get; init; } 

    /// <summary>
    /// 비지니스 유닛
    /// </summary>
    // ReSharper disable once CollectionNeverQueried.Global
    public List<ResponseBusinessUnit> BusinessUnits { get; set; } = new List<ResponseBusinessUnit>();
}

File: BudgetBringerV2/src/models/responses/budgets/response-country-business-manager.ts
import type { ResponseBusinessUnit } from '@/models/responses/budgets/response-business-unit'
import type { ResponseCommonWriter } from '@/models/responses/response-common-writer'

/**
 * CBM 관리 응답 모델
 */
export interface ResponseCountryBusinessManager extends ResponseCommonWriter {
  /**
   * 아이디
   */
  id: string;
  /**
   * 오너명
   */
  name: string;
  /**
   * CBM 이 소유한 비지니스 유닛
   */
  businessUnits: Array<ResponseBusinessUnit>;
  /**
   * Sequence
   */
  sequence: number;
}
